src/subsplit_dag_node.hpp

```diff
@@ -119,7 +123,7 @@ class SubsplitDAGNode {
   // dag_nodes_.
   size_t id_;
   // Node bitset subsplit clades.
-  const Bitset subsplit_;
+  Bitset subsplit_;
```

Comment:
Do we really have to make these mutable? Why not make a new one when we need to change the subsplit?